src/Interpreters/Cache/WriteBufferToFileSegment.h

```diff
@@ -17,6 +17,8 @@ class WriteBufferToFileSegment : public WriteBufferFromFileDecorator, public IRe
 
     void nextImpl() override;
 
+    ~WriteBufferToFileSegment() override = default;
```

Comment:
What does it change?

Reply:
I assume nothing :) https://github.com/ClickHouse/ClickHouse/pull/63346#discussion_r1589299406